### ISSUE TITLE
Cannot install gem ruby -v 2,2,3

### ISSUE THREAD

**Issue by sergio-xga** [cparedes/opsview_rest#12]
Same error when trying to install the gem from github source

```
$sudo gem install opsview_rest
Building native extensions.  This could take a while...
ERROR:  Error installing opsview_rest:
    ERROR: Failed to build gem native extension.

    current directory: /usr/local/lib/ruby/gems/2.2.0/gems/json-1.7.7/ext/json/ext/generator
/usr/local/bin/ruby -r ./siteconf20160422-3961-yyywhx.rb extconf.rb
creating Makefile

current directory: /usr/local/lib/ruby/gems/2.2.0/gems/json-1.7.7/ext/json/ext/generator
make "DESTDIR=" clean

current directory: /usr/local/lib/ruby/gems/2.2.0/gems/json-1.7.7/ext/json/ext/generator
make "DESTDIR="
compiling generator.c
In file included from generator.c:1:0:
../fbuffer/fbuffer.h: In function ‘fbuffer_to_s’:
../fbuffer/fbuffer.h:175:47: error: macro "rb_str_new" requires 2 arguments, but only 1 given
     VALUE result = rb_str_new(FBUFFER_PAIR(fb));
                                               ^
../fbuffer/fbuffer.h:175:20: warning: initialization makes integer from pointer without a cast [enabled by default]
     VALUE result = rb_str_new(FBUFFER_PAIR(fb));
                    ^
make: *** [generator.o] Error 1

make failed, exit code 2

Gem files will remain installed in /usr/local/lib/ruby/gems/2.2.0/gems/json-1.7.7 for inspection.
Results logged to /usr/local/lib/ruby/gems/2.2.0/extensions/x86_64-linux/2.2.0-static/json-1.7.7/gem_make.out
```

**Comment by damacus** [comment]
Any update on this pr? I'm looking at using this gem for a chef cookbook and chef uses ruby 2.2.3

**Comment by damacus** [comment]
This should have now been fixed by #14

**Comment by cparedes** [comment]
Fixed by #14
